packages/client/src/scenes/worldScene.ts

```diff
@@ -105,7 +105,17 @@ export class WorldScene extends Phaser.Scene {
       frameRate: 6,
       repeat: -1
     });
-
+    // this.anims.create({
+    //     key: `volacano`,
+    //     frames: this.anims.generateFrameNames('global_atlas', {
+    //       start: 1,
+    //       end: 8,
+    //       prefix: `volcano-`
+    //       //suffix: '.png'
+    //     }),
+    //     frameRate: 6,
+    //     repeat: -1
+    //   });
```

Comment:
We should just delete this code instead of commenting it out (it adds noise to the code base)